src/MeshKernelNET/Native/MeshKernelDll.cs

```diff
@@ -1433,6 +1433,32 @@ static MeshKernelDll()
                                                               [In] int minimumNumSamples,
                                                               [In] ref MeshRefinementParametersNative meshRefinementParameters);
 
+        /// <summary>
+        /// Rotates a mesh2d about a given point by a given angle
+        /// </summary>
+        /// <param name="meshKernelId">Id of the mesh state</param>
+        /// <param name="centreX">>X-coordinate of the centre of rotation</param>
+        /// <param name="centreY">Y-coordinate of the centre of rotation></param>
+        /// <param name="angle">Angle of rotation</param>
```

Comment:
suggest to document that angle is in radians or degrees

Reply:
Degrees. will add it.